torch/_export/serde/serialize.py

```diff
@@ -615,25 +616,25 @@ def serialize_optional_tensor_args(a):
                 return Argument.create(
                     as_optional_tensors=list(map(serialize_optional_tensor_args, arg))
                 )
-            elif all(isinstance(a, (inductor_ir.InputBuffer, inductor_ir.ComputedBuffer)) for a in arg):
+            elif all(isinstance(a, inductor_tensor_buffers) for a in arg):
                 # list of tensors
                 return Argument.create(
                     as_tensors=[TensorArgument(name=a.name) for a in arg],
                 )
-            elif all(isinstance(a, (inductor_ir.InputBuffer, inductor_ir.ComputedBuffer, type(None))) for a in arg):
+            elif all(isinstance(a, (*inductor_tensor_buffers, type(None))) for a in arg):
                 # list of optional tensors
                 def serialize_optional_tensor_args(a):
                     if a is None:
                         return OptionalTensorArgument.create(as_none=())
-                    elif isinstance(a, torch._inductor.ir.InputBuffer):
+                    elif isinstance(a, inductor_tensor_buffers):
```

Comment:
The use of ```inductor_tensor_buffers``` changes the old semantics, where only ```InputBuffer``` is supported. Just want to double check if it's a desired behavior. Thanks.

Reply:
yes, this is intentional.